tests/tests_unit/graph/test_loaders/test_dms_loader.py

```diff
@@ -15,16 +16,36 @@ def test_metadata_as_json_filed():
 
     importer = InferenceImporter.from_graph_store(store)
 
-    rules = ImporterPipeline.verify(importer)
-    store.add_rules(rules)
+    info_rules = ImporterPipeline.verify(importer)
+    dms_rules = InformationToDMS().transform(info_rules).rules
+
```

Comment:
This test simulates the process of updating data model on both ends but still being able to get what you are expecting